Core/Core/Helpers/BinaryFormatter.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using System.IO;
+using System.Text;
+
+namespace Speckle.Core.Helpers;
+
+/// <summary>
+/// Mimics the behaviour of System.Runtime.Serialization.Formatters.Binary.BinaryFormatter for compatibility with Dotnet 8
+/// </summary>
+public static class BinaryFormatter
+{
+  /// <summary>
+  /// Leading bytes on a BinaryFormatter as defined here: 
+  /// https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-nrbf/a7e578d3-400a-4249-9424-7529d10d1b3c 
+  /// and here: https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-nrbf/eb503ca5-e1f6-4271-a7ee-c4ca38d07996
+  /// </summary>
+  private static readonly byte[] s_prefix = new byte[] { 
+      0,                   // RecordTypeEnum (always 0)
+      1,   0,   0,   0,    // Root ID, Id of the incoming string (see below)
+    255, 255, 255, 255,    // Header ID
+      1,   0,   0,   0,    // Major Version
+      0,   0,   0,   0,    // Minor Version
+      6,                   // RecordTypeEnum for string - must be 6
+      1,   0,   0,   0     // Id of the upcoming string, 1 as there is no other string.
+  };
+  /// <summary>
+  /// Indicates end of file
+  /// </summary>
+  private const byte ENDOFFILE = 11;
+
+  /// <summary>
+  /// Mimics the behaviour of System.Runtime.Serialization.Formatters.Binary.BinaryFormatter().Serialize() when applied to a single string record.
+  /// </summary>
+  /// <param name="ms">Memory stream to append to</param>
+  /// <param name="value">String to serialise</param>
+  public static void SerialiseString(MemoryStream ms, string value)
```

Comment:
per JR's query "guessing there's not much we can do about these statics atm? extensions methods maybe an option?" 

happy to follow any pattern suggested - my goal was to err as close to the original serialiser as I could for familiaritys sake.